### ISSUE TITLE
random-1.2 bounds

### ISSUE THREAD

**Issue by lehins** [LambdaHack/LambdaHack#244]
It looks like LambdaHack-0.10 no longer uses `random` package, but would it be possible to make a revision for `LambdaHack-0.9.5` with relaxed bounds: `random >= 1.0 && < 1.3` instead of `random == 1.1.*`
 
Because `LambdaHack-0.10` is still not on hackage we'll have to disable it on stackage otherwise: commercialhaskell/stackage#5474

**Comment by Mikolaj** [comment]
Hi! Thank you for the heads up. Yes, please disable it for now. v0.10 will take a bit more time and v0.9 does need the old random.

**Comment by Mikolaj** [comment]
Hi @lehins! https://hackage.haskell.org/package/LambdaHack-0.10.2.0 and https://hackage.haskell.org/package/Allure-0.10.2.0 solve the problem. Would stackage pick up the new versions or should we enable something manually?

**Comment by lehins** [comment]
@Mikolaj Awesome! Nothing needs to be done, stackage will pick it up since `random` hasn't been added to stackage yet, so `LambdHack` was no disabled yet and thanks to then new version it doesn't need to be :wink:

**Comment by Mikolaj** [comment]
@lehins: amazing, thank you!

**Comment by lehins** [comment]
Spoke too soon. LambdaHack seems to depend on `int-cast` which was disabled because some incompatibility with ghc-8.10 (probably just bounds issues)

It is not a blocker for `random` because `LambdaHack` no longer depends on it. 

However, getting LambdaHack on stackage will be a bit problematic, since int-cast is maintained by HVR and he is no longer  active in OSS

**Comment by Mikolaj** [comment]
Really? The last revision of int-cast seems fine: https://matrix.hackage.haskell.org/#/package/int-cast

**Comment by lehins** [comment]
Not quite sure why `int-cast` is disabled, I just relayed what was in the [comment](https://github.com/commercialhaskell/stackage/blob/18c6961b705fca0fce72babd4eabbdc2ff02a20a/build-constraints.yaml#L4320)

However when I try to build it with latest nightly snapshot [bounds for QuickCheck are out of wack](https://github.com/haskell-hvr/int-cast/blob/a25250b7c69af61166f33b8fc692f7be2bb0bd52/int-cast.cabal#L47), so disabling the test suite does seem to help with ghc 8.10. However from what I know stackage team is trying to get ghc-9 (https://github.com/commercialhaskell/stackage/pull/5871#issuecomment-782721575) in and int-cast has the upper bound that makes it unbuildable. Maybe hackage trustees will be able to help with this. For now I've submitted a PR: commercialhaskell/stackage#5978
